include/CL/opencl.hpp

```diff
@@ -392,6 +397,14 @@
 #ifndef CL_HPP_
 #define CL_HPP_
 
+#ifdef CL_HPP_OPENCL_API_WRAPPER
+#define _CL(name) CL_HPP_OPENCL_API_WRAPPER(name)
+#define _CL_PTR(name) CL_HPP_OPENCL_API_WRAPPER(name)
+#else
+#define _CL(name) ::name
+#define _CL_PTR(name) &::name
```

Comment:
Do we need both `_CL` and `_CL_PTR`?  I can't figure out why `_CL_PTR` is needed, and it seems like we could use `_CL` in all cases.

If we do need `_CL_PTR` for some reason, should there be a separate wrapper macro for it as well?  I'd prefer not to do this, and would much rather eliminate `_CL_PTR`, but we should stay symmetric regardless.

One file nit: identifiers beginning with an underscore and a capital letter are considered reserved IIRC, so although I appreciate the brevity we should use a different spelling than `_CL`.

Reply:
For some reason I was thinking that a `&` is needed when taking a function pointer, but I think that's only the case for pointer-to-members, not for normal function pointers, so using `_CL` everywhere should work fine, yes.

Also, I'll rename `_CL` to `CL_` which is not reserved.

Reply:
Ok, I've updated the PR.